src/openApi/openApiWrapper.ts

```diff
@@ -6,6 +6,7 @@ import { Exception } from '../exceptions/exception';
 import { safeJwtCanonicalIdParse, serializeObject } from '../util';
 import { orionCorrelationIdRoot } from '../shared';
 import { OpenApiModel } from './openApiModel';
+import { redactSecret } from '../logger/logger';
```

Comment:
I'd move `redactSecret` e.g. to `util` if it's used not only for logging

Reply:
done